pipelines/enterprise-contract.yaml

```diff
@@ -75,6 +83,10 @@ spec:
           value: "true"
         - name: TIMEOUT
           value: "$(params.TIMEOUT)"
+        - name: CA_TRUST_CONFIGMAP_NAME
```

Comment:
The `ec-task-bundle` needs to be updated to include a version of the EC Task that accepts these parameters. The current reference does not:

```bash
🐚 tkn bundle list -o yaml quay.io/enterprise-contract/ec-task-bundle:snapshot@sha256:294b14582fa0e44f42c7e0651915ca67425488527fa7d9ecb49c3974ede028fc | yq .spec.params[].name
*Warning*: This is an experimental command, it's usage and behavior can change in the next release(s)
DEFINITION
POLICY_SOURCE
NAMESPACE
POLICY_LIB
POLICY_DATA
HOMEDIR
```

Reply:
@lcarva If I understand correctly, that command returns the `.spec.params[].name` of the last spec it finds, which is the spec of the `verify-definition` task.
I don't think this is actually a valid yaml so yq might not be the tool to process it.

Looking at the output, without yq, I do see those parameters under the `verify-enterprise-contract` task.

Reply:
Thanks @lcarva and @yftacherzog!

Reply:
FYI https://github.com/tektoncd/cli/issues/2091